core/sql/regress/executor/TEST002

```diff
@@ -661,7 +661,9 @@ left outer join (
   ) T) T) T) T) T) T
 on (t002t6.c = T.a)
 ;
+
 control query shape merge_join(cut,cut);
+cqd PCODE_OPT_LEVEL '0';
```

Comment:
Yes, I'll file a JIRA. In the process ....

On Mon, Sep 21, 2015 at 10:59 AM, DaveBirdsall <notifications@github.com>
wrote:

> In core/sql/regress/executor/TEST002
> <https://github.com/apache/incubator-trafodion/pull/87#discussion_r39988524>
> :
>
> >  control query shape merge_join(cut,cut);
> > +cqd PCODE_OPT_LEVEL '0';
>
> This is worrisome actually. Is there a follow-on JIRA to fix the Pcode
> problems?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-trafodion/pull/87/files#r39988524>.
>



-- 
Regards, --Qifan